### DISCUSSION TITLE
Install Janssen one local cloud Multipass

### DISCUSSION THREAD

**Discussion by BainMcKay** [JanssenProject/jans#7360] [Q&A]
Installing Janssen on multipass/ubuntu 20.04  on Mac Studio Ultra host running Ventura
* Standard install
* ``apt update (did not use `apt upgrade`)
* package issue

root@localgluu-mp2012:/etc/site_certificates/localgluu-mp2012.kayviumdev.com# cd /tmp
sha256sum -c jans_1.0.20.ubuntu22.04_amd64.deb.sha256sum
jans_1.0.20.ubuntu22.04_amd64.deb: OK
root@localgluu-mp2012:/tmp# sudo apt install  ./jans_1.0.20.ubuntu22.04_amd64.deb
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'jans:amd64' instead of './jans_1.0.20.ubuntu22.04_amd64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 jans:amd64 : Depends: libc6:amd64 (>= 2.17) but it is not installable
              Depends: libgcc-s1:amd64 (>= 4.2) but it is not installable
              Depends: apache2:amd64 but it is not installable
              Depends: rsyslog:amd64 but it is not installable
              Depends: python3-urllib3:amd64 but it is not installable
              Depends: python3-certifi:amd64 but it is not installable
              Depends: python3-ldap3:amd64 but it is not installable
              Depends: python3-requests:amd64 but it is not installable
              Depends: python3-ruamel.yaml:amd64 but it is not installable
              Depends: python3-pymysql:amd64 but it is not installable
              Depends: python3-cryptography:amd64 but it is not installable
              Depends: python3-psycopg2:amd64 but it is not installable
E: Unable to correct problems, you have held broken packages.

**Comment by BainMcKay** [comment]
running `apt upgrade` made no difference. This is new behaviour. from a couple of weeks ago.

from this I assume Janssen if not installable on an m1 Mac Studio Ultra under Multipass/ubuntu 22.04

Next I will try m1 Mac Studio Ultra under Multipass/ubuntu 20.04

**Comment by BainMcKay** [comment]
I tried instal on m1 Mac Studio Ultra under Multipass/ubuntu 20.04
* No success, with `apt update` and `apt upgrade1`
* * So Include that Janssen is not installable in multipass/ubuntu  on a  m1 Mac Studio Ultra at this time with the current installation instructions 

That leaves: 
* Digital Ocean, where it would not install with OpenDJ but did with MySQL as the IDSA. Except it throws an exception on login OpenID client authentication handshake at login.
* *AWS EC2 where is has a socket socket issue with openDJ, no matter what I do, and MySQL where it is a smooth simple installation but I can't get the browser to recognize the TUI interface for validation. Might this have something good with the Tenant Interface? Perhaps I need to open up additional ports in the tenant interface

**Comment by ossdhaval** [comment]
@BainMcKay 

With the current stable release, you can use [Docker compose](https://docs.jans.io/stable/janssen-server/install/docker-install/compose/) to install Janssen.
